pandas/core/indexes/base.py

```diff
@@ -4246,20 +4246,67 @@ def _add_logical_methods(cls):
         """ add in logical methods """
 
         _doc = """
-
         %(desc)s
 
         Parameters
         ----------
-        All arguments to numpy.%(outname)s are accepted.
+        *args
+            These parameters will be passed to numpy.%(outname)s.
+        **kwargs
+            These parameters will be passed to numpy.%(outname)s.
 
         Returns
         -------
         %(outname)s : bool or array_like (if axis is specified)
             A single element array_like may be converted to bool."""
 
+        _index_shared_docs['index_all'] = """
+
+        See Also
+        --------
+        pandas.Index.any : Return whether any element is True.
+
+        Notes
+        -----
+        Not a Number (NaN), positive infinity and negative infinity
+        evaluate to True because these are not equal to zero.
+
+        Examples
+        --------
+        >>> index = pd.Index([1,2,3])
```

Comment:
PEP8: space after commas

Reply:
################################################################################
######################### Docstring (pandas.Index.all) #########################
################################################################################

Return whether all elements are True.

Parameters
----------
*args
    These parameters will be passed to numpy.all.
**kwargs
    These parameters will be passed to numpy.all.

Returns
-------
all : bool or array_like (if axis is specified)
    A single element array_like may be converted to bool.

        See Also
        --------
        pandas.Index.any : Return whether any element is True.

        Notes
        -----
        Not a Number (NaN), positive infinity and negative infinity
        evaluate to True because these are not equal to zero.

        Examples
        --------
        >>> index = pd.Index([1, 2, 3])
        >>> index.all()
        True

        >>> index = pd.Index([0, 1, 2])
        >>> index.all()
        False

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
        No extended summary found
        Errors in parameters section
                Parameters {'kwargs', 'args'} not documented
                Unknown parameters {'**kwargs', '*args'}
                Parameter "*args" has no type
                Parameter "**kwargs" has no type